src/pages/pretest-checks/pretest-checks.html

```diff
@@ -3,112 +3,140 @@
-            </div>
-          </button>
-        </ion-col>
-      </ion-row>
-      <div class="mes-divider"></div>
-      <ion-row>
-        <ion-col col-4 class="mes-header-md">
-          Tell me question
-        </ion-col>
-        <ion-col offset-3 col-5>
-          <select-button [onClick]="showTellMeOptions.bind(this)" [selectedText]="tellMeQuestionId"></select-button>
-        </ion-col>
-      </ion-row>
-      <div class="mes-divider"></div>
-      <ion-row class="height-auto">
-        <ion-col col-4 class="mes-header-md">
-          Accompanied by
-        </ion-col>
-      </ion-row>
-      <ion-row class="center-text">
-        <ion-col col-4 class="mes-header-md">
-          ADI
-          <input type="checkbox" class="mes-checkbox">
-        </ion-col>
-        <ion-col col-4 class="mes-header-md">
-          Sup
-          <input type="checkbox" class="mes-checkbox">
-        </ion-col>
-        <ion-col col-4 class="mes-header-md">
-          Other
-          <input type="checkbox" class="mes-checkbox">
-        </ion-col>
-      </ion-row>
-    </ion-card-content>
-  </ion-card>
+  <form #form="ngForm" (ngSubmit)="gotoDL25(form)" novalidate>
+    <ion-card class="mes-card">
+      <ion-card-header class="mes-breadcrumbs">
+        <span class="mes-header-md mes-breadcrumb-link" navPop>
+          <ion-icon name="arrow-back"></ion-icon>
+          View Candidate /
+        </span>
+        <span padding-left class="mes-header-md">Begin Test</span>
+      </ion-card-header>
+      <ion-card-content>
+        <ion-row>
+          <ion-col col-4 class="mes-header-md">
+            Eyesight*
+          </ion-col>
+          <ion-col col-4 class="mes-header-md center-text">
+            Completed
+            <input #isEyesightCompleted="ngModel" name="isEyesightCompleted" [(ngModel)]="preCheck.isEyesightCompleted" type="checkbox"
+              required class="mes-checkbox">
+          </ion-col>
+          <ion-col offset-1 col-3 class="center-text">
+            <button class="mes-header-md eyesight-record-serious-fault-button" (click)="showEyesightFaultRecordingModal()">
+              <span>S</span>
+              <div>
+                <input class="checkbox" type="checkbox" [(ngModel)]="eyeSightSeriousFault" [ngModelOptions]="{standalone: true}">
+              </div>
+            </button>
+          </ion-col>
+        </ion-row>
+        <p [hidden]="isEyesightCompleted.valid || (isEyesightCompleted.pristine && !form.submitted)" class="warning-text">
+          Eyesight is required.
+        </p>
+        <div class="mes-divider"></div>
+        <ion-row>
+          <ion-col col-4 class="mes-header-md">
+            Tell me question*
+          </ion-col>
+          <ion-col offset-3 col-5>
+            <select-button required #tellMeQuestion="ngModel" [(ngModel)]="preCheck.tellMeQuestionId" name="tellMeQuestionId" [onClick]="showTellMeOptions.bind(this)"
```

Comment:
How come we are using `[onClick]` and `.bind` and not `(click)="showTellMeOptions()"`?